fastlane/Fastfile

```diff
@@ -1,26 +1,121 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/. 
-
-fastlane_version "1.39.0"
-
-default_platform :ios
-
-# Assume that the builder has checked out the build-tools repo manually
-#import_build_tools(
-#  url: 'git@github.com:mozilla/firefox-ios-build-tools.git',
-#  clone_folder: "../build-tools",
-#)
-
-#move all the build tools to the right places
-clone_folder = "../../build-tools"
-sh("rsync -a #{clone_folder}/scripts ../")
-sh("rsync -a #{clone_folder}/fastlane/Appfile Appfile")
-sh("rsync -a #{clone_folder}/fastlane/Snapfile Snapfile")
-sh("rsync -a #{clone_folder}/fastlane/SnapshotHelper.swift SnapshotHelper.swift")
-sh("rsync -a #{clone_folder}/fastlane/scripts .")
-sh("rsync -a #{clone_folder}/fastlane/frames .")
-sh("rsync -a #{clone_folder}/fastlane/templates .")
-
-#import the fastfile
-import "#{clone_folder}/fastlane/BaseFastfile"
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Continous deployment for Cliqz iOS"
+	
+  # Configuring Sentry DSN
+  sh("cd .. && plutil -replace SentryDSN -string $SentryDSN Client/Info.plist")
+  	
+  # Building ReactNative Entension
+  sh("cd .. && npm run bundle")
+  
+  lane :beta do
+    # add actions here: https://docs.fastlane.tools/actions
+    
+  	create_keychain(
```

Comment:
please fix the indentation in the file, almost every block is formated in different way